lit/command/CommitCommand.py

```diff
@@ -1,59 +1,57 @@
-from lit.file.StringManager import StringManager
-from lit.file.SettingsManager import SettingsManager
 from lit.file.JSONSerializer import JSONSerializer
-import hashlib
+from lit.strings_holder import StringsHolder
 
 
 class CommitCommand(BaseCommand):
-    __COMMAND_COMMIT_NAME_KEY = 'COMMAND_COMMIT_NAME'
-    __COMMAND_COMMIT_HELP_KEY = 'COMMAND_COMMIT_HELP'
-    __COMMAND_COMMIT_ARGUMENT_MESSAGE_NAME_KEY = 'COMMAND_COMMIT_ARGUMENT_MESSAGE_NAME'
-    __COMMAND_COMMIT_ARGUMENT_MESSAGE_HELP_KEY = 'COMMAND_COMMIT_ARGUMENT_MESSAGE_HELP'
-
     def __init__(self):
-        name = StringManager.get_string(self.__COMMAND_COMMIT_NAME_KEY)
-        help_message = StringManager.get_string(self.__COMMAND_COMMIT_HELP_KEY)
+        name = StringsHolder.Commands.Commit.NAME
+        help_message = StringsHolder.Commands.Commit.HELP
         arguments = [
             CommandArgument(
-                name=StringManager.get_string(self.__COMMAND_COMMIT_ARGUMENT_MESSAGE_NAME_KEY),
+                name=StringsHolder.Commands.Commit.Arguments.MSG_NAME,
                 type=str,
-                help=StringManager.get_string(self.__COMMAND_COMMIT_ARGUMENT_MESSAGE_HELP_KEY)
+                help=StringsHolder.Commands.Commit.Arguments.MSG_HELP
             ),
         ]
         super().__init__(name, help_message, arguments)
 
     def run(self, **args):
 
-        serializer_tracked = JSONSerializer(SettingsManager.get_var_value('TRACKED_FILE_PATH'))
+        serializer_tracked = JSONSerializer(StringsHolder.TrackedFileSettings.PATH)
         tracked = serializer_tracked.read_all_items()
 
-        file_count = len(os.listdir(SettingsManager.get_var_value('COMMIT_FILES_IN_COMMIT_DIR')))
-        with ZipFile(SettingsManager.get_var_value('COMMIT_ZIP_FILE_NAME') + str(file_count) +
-                             SettingsManager.get_var_value('COMMIT_ZIP_EXTENCION'), 'w') as myzip:
-            for file in tracked['files']:
+        file_count = len(os.listdir(StringsHolder.CommitSettings.DIR_PATH.value))
+        with ZipFile(StringsHolder.CommitSettings.ZIP_FILE_NAME + str(file_count) +
+                     StringsHolder.CommitSettings.ZIP_EXTENSION, 'w') as myzip:
+            files_key = StringsHolder.TrackedFileSettings.FILES_KEY
+            for file in tracked[files_key]:
                 myzip.write(file)
 
         myzip_hash = self.get_file_hash(myzip.filename)
+        message = StringsHolder.Commands.Commit.Arguments.MSG_NAME.value
         commit = {
-            SettingsManager.get_var_value('COMMIT_USER'): 'WIP',
-            SettingsManager.get_var_value('COMMIT_LONG_HASH'): myzip_hash,
-            SettingsManager.get_var_value('COMMIT_SHORT_HASH'): myzip_hash[:10],
-            SettingsManager.get_var_value('COMMIT_DATETIME'): str(datetime.utcnow()),
-            SettingsManager.get_var_value('COMMIT_COMMENT'): args['message'],
+            StringsHolder.CommitSettings.USER: 'WIP',
```

Comment:
This way of storing strings is not suitable in this case. You should be able to change VCS user's name, but in this case the user's name is hardcoded in python source file and cannot be changed except modifying the sources.

Reply:
Let me solve it in another branch...